plans/scripts/install-podman.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -eu
+
+# Uninstall a preinstalled Podman version to ensure the desired version will be installed.
+sudo dnf remove -y podman
+
+# Construct the download URL for the specific Podman version.
+COMPOSE_VERSION="fc$(echo "$COMPOSE" | cut -d '-' -f 2)"
+CUSTOM_PODMAN_URL="https://kojipkgs.fedoraproject.org//packages/podman/${PODMAN_VERSION}/1.${COMPOSE_VERSION}/${ARCH}/podman-${PODMAN_VERSION}-1.${COMPOSE_VERSION}.${ARCH}.rpm"
+
+# If the latest stable Podman version is requested, install it from the official Fedora repository.
+# Otherwise, download and install the specific RPM package.
+if [[ "$PODMAN_VERSION" == "latest" ]]; then
+    sudo dnf install -y podman --disablerepo=testing-farm-tag-repository
+    PODMAN_VERSION="$(curl -s https://api.github.com/repos/containers/podman/releases/latest | jq -r .tag_name | sed 's/^v//')"
```

Comment:
you might encounter github api rate limit when not using github_token. But let's see.